third_party/intellij-plugins-dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceDebugProcessZ.java

```diff
@@ -110,11 +110,6 @@ public boolean isRemoteDebug() {
     return remoteDebug;
   }
 
-  @NotNull
-  public ObservatoryConnector getConnector() {
```

Comment:
Just to confirm, there are no callers, and we're not overriding (and changing behavior) from the parent class?